Source/Dafny/DafnyOptions.cs

```diff
@@ -1082,6 +1082,8 @@ is printed.
 
 /warnShadowing  Emits a warning if the name of a declared variable caused another variable
     to be shadowed
+/warnMissingConstructorParenthesis Emits a warning when a constructor name is not followed 
```

Comment:
I think we should be more specific, "when a constructor name in a case pattern is not followed by parentheses", note the plural "parentheses" as well.